apps/molgenis-components/src/components/layout/GTag.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div></div>
+</template>
+
+<script>
+import { setOptions, bootstrap } from "vue-gtag";
+export default {
+  name: "GTag",
+  props: {
+    tagId: {
+      type: String,
+      required: true,
+    },
+  },
+  methods: {
+    enablePlugin(tagId) {
+      setOptions({
+        config: { id: tagId },
+      });
+
+      bootstrap().then((gtag) => {
+        console.log("start analytics for tagId: " + tagId);
```

Comment:
do you want a console.log in production? not very professional.

Reply:
yes would